addons/netfox/properties/property-entry.gd

```diff
@@ -28,6 +27,5 @@ static func parse(root: Node, path: String) -> PropertyEntry:
 	var result = PropertyEntry.new()
 	result.node = root.get_node(NodePath(path))
 	result.property = path.erase(0, path.find(":") + 1)
-	result.type = typeof(result.get_value())
```

Comment:
Should I make a PR for caching the type of the property entry?

Reply:
Not yet, that would need a separate discussion on why that would be beneficial.